src/Lecture3.hs

```diff
@@ -89,7 +93,8 @@ weekday to the second.
 >>> daysTo Friday Wednesday
 5
 -}
-daysTo = error "TODO"
+daysTo :: (Eq a, Bounded a, Enum a) => a -> a -> Int
+daysTo a b = length $ takeWhile (/= b) $ iterate next a
```

Comment:
That's a fun implementation 😅 
But it's a bit inefficient. I think, it would be better to use a couple of arithmetic operations here 🙂 

Also, there's no need to have this function so polymorphic. It's still called `daysTo` and not every enumeration has days.